### ISSUE TITLE
queries not working after upgrading from 0.5.3 to 1.0.6

### ISSUE THREAD

**Issue by sourceful-mia** [7nohe/openapi-react-query-codegen#75]
Very cool repo! Love the work that's being done.

I have an issue with version 1.0 when trying to codegen for a very simple OpenAPI Spec.

When doing codegen with v0.5.3, I get content in my generated `index.ts` (eg: `useDefaultServiceGetProjects`)
However, when doing codegen with v1.0.6, none of the files contain the above hook.

My spec is generated with `tsoa` and valid according to https://editor.swagger.io/

Is something missing from my spec, or is there a bug in this repo?

Here's the spec
```
openapi: 3.0.0
components:
  examples: {}
  headers: {}
  parameters: {}
  requestBodies: {}
  responses: {}
  schemas:
    Project:
      properties:
        id:
          type: string
        name:
          type: string
        description:
          type: string
      required:
        - id
        - name
        - description
      type: object
      additionalProperties: false
    ProjectCreateParams:
      properties:
        name:
          type: string
      required:
        - name
      type: object
      additionalProperties: false
  securitySchemes: {}
info:
  title: projects-api
  version: 1.0.0
  description: Projects backend for frontend
  license:
    name: ISC
  contact: {}
paths:
  /v1/projects:
    get:
      operationId: GetProjects
      responses:
        "200":
          description: Ok
          content:
            application/json:
              schema:
                items:
                  $ref: "#/components/schemas/Project"
                type: array
      security: []
      parameters: []
    post:
      operationId: CreateProject
      responses:
        "201":
          description: Created
      security: []
      parameters: []
      requestBody:
        required: true
        content:
          application/json:
            schema:
              $ref: "#/components/schemas/ProjectCreateParams"
  /v1/projects/{id}:
    get:
      operationId: GetProject
      responses:
        "200":
          description: Ok
          content:
            application/json:
              schema:
                $ref: "#/components/schemas/Project"
      security: []
      parameters:
        - in: path
          name: id
          required: true
          schema:
            type: string
servers:
  - url: /
```

**Comment by Jakob-Espersen** [comment]
I had a similar problem. For me upgrading my typescript version to the latest worked.

**Comment by sourceful-mia** [comment]
> I had a similar problem. For me upgrading my typescript version to the latest worked.

I'm on typescript 5.4.5, which is the latest version.

**Comment by seriouslag** [comment]
I was able to generate the service files and hooks using:
```
node: 21.x
typescript: 5.4.3
openapi-react-query-codegen: 1.0.6
```
with the spec file you provided

If you could provide:
- an example repo,
- the command you used to call openapi-react-query-codegen

I would gladly take a look further into your problem.

**Comment by seriouslag** [comment]
> The prettier implementation breaks the queries generation.
> 
> There's a reproduction: https://stackblitz.com/edit/tanstack-query-e7gu7z
> 
> 
> 
> Remove `prettier` from the dependencies, run `npm run codegen` and queries are back.
> 
> 
> 
> edit: The workaround is to add generated files to the `.prettierignore`

I was able to generate the code using your example with prettier installed and no .prettierignore file. I will have to test more. If you have time to test more I would love more feedback to figure this out

**Comment by sourceful-mia** [comment]
@seriouslag https://github.com/sourceful-mia/rep-openapi-react-query
Here's my rep case repo.
if you run 
```
nvm install && nvm use
npm install
npm run codegen
```
then it works fine.

If you then run
```
npm i prettier
npm run codegen
```
then it stops working. I think @gynekolog might be right about prettier.

**Comment by seriouslag** [comment]
I reproduced the issue, and I will debug it. Thank you

**Comment by seriouslag** [comment]
I debugged the issue and pushed the fix to #74 

Prettier was switching quotes with double quotes and breaking our comparisons.
Changed our comparison to not care about quotes.

**Comment by seriouslag** [comment]
Released a test version with this patch under: [@seriouslag/openapi-react-query-codegen@next](https://www.npmjs.com/package/@seriouslag/openapi-react-query-codegen/v/next)
Until we merge the patches into this repo 😀
